contrib/msggen/msggen/schema.json

```diff
@@ -29534,272 +29534,6 @@
       "Main web site: <https://github.com/ElementsProject/lightning>"
     ]
   },
-  "lightning-sql.json": {
-    "$schema": "../rpc-schema-draft.json",
-    "type": "object",
```

Comment:
Mayube fold last commit into previous one (or wherever it is needed?)

Reply:
I squashed two commits related to lightning-sql into one commit `docs: Updates script and fixed test to generate lightning-sql`. I hope that fixed this one too. If not, could you please specify which commit you want folded?